frontend/src/app/api/applications/start/route.ts

```diff
@@ -0,0 +1,36 @@
+import { ApiRequestError, readError, UnauthorizedError } from "src/errors";
+import { handleStartApplication } from "src/services/fetch/fetchers/applicationFetcher";
+
+const COMPETITION_ID = "fd7f5921-9585-48a5-ab0f-e726f4d1ef94";
+
+export const POST = async () => {
+  try {
+    // TODO: add session support;
+    const session = { token: true };
```

Comment:
can't we use the session from `getSession`?

Reply:
We aren't doing logged in users so it is out of scope.